apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.IntegrationTests/ErrorReporting/ExceptionLoggerTest.cs

```diff
@@ -42,12 +43,17 @@ public ExceptionLoggerTest()
         }
 
         [Fact]
-        public async Task ErrorReportingTest()
+        public Task ErrorReportingTest_ErrorReporting() => ErrorReportingTest<ReportToErrorReportingApplication>();
+
+        [Fact]
+        public Task ErrorReportingTest_Logging() => ErrorReportingTest<ReportToLoggingApplication>();
+
+        public async Task ErrorReportingTest<T>()
```

Comment:
Possibly make these helper methods private and move them all into one place, to make it clear which bits are tests and which are helpers? I initially wondered whether this just accidentally didn't have a `[Fact]` attribute.